src/main/scala/io/sdkman/changelogs/KcctlMigrations.scala

```diff
@@ -0,0 +1,22 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "064")
+class KcctlMigrations {
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_kcctl_candidate",
+    author = "gunnarmorling"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "kcctl",
+      name = "kcctl",
+      description =
+        "kcctl - A modern and intuitive command line client for Kafka Connect.",
```

Comment:
I know that I'm a PITA :smile: but can you make this a bit more detailed, please? The description is the blab displayed for your candidate when you type `sdk list` or when you visit https://sdkman.io/sdks.

I like what you have on your GitHub page. Could you perhaps adapt that as your description?

Reply:
Hhow about that:

> kcctl - a modern and intuitive command-line client for Kafka Connect. Relying on the idioms and semantics of kubectl, it allows you to register and examine connectors, delete them, restart them, etc.

Reply:
Went ahead and pushed that update. Let me know if you think it still needs further refinement.

Reply:
Excellent, I made one minor tweak if you don't mind.

Reply:
Perfect, thx!